embedchain/apps/PersonApp.py

```diff
@@ -22,6 +22,23 @@ def __init__(self, person, config: BaseAppConfig = None):
         self.person_prompt = f"You are {person}. Whatever you say, you will always say in {person} style."  # noqa:E501
         super().__init__(config)
 
+    def add_person_template_to_config(self, default_prompt: str, config: ChatConfig = None):
+
+        self.template = Template(self.person_prompt + " " + default_prompt)
```

Comment:
Maybe it's easier to understand if we turn this on its head:

```python
if config:
    if config.template:
        # Add person prompt to custom user template
        config.template = Template(self.person_prompt + " " + config.template.template)
    else:
        # If no user template is present, use person prompt with the default template
        config.template = self.template
else:
    # if no config is present at all, initialize the config with person prompt and default template
    config = QueryConfig(
                template=self.template,
            )
```

Reply:
updated